src/api/app/controllers/webui/package_controller.rb

```diff
@@ -663,6 +663,16 @@ def remove
     end
   end
 
+  def trigger_services
+    begin
+      Suse::Backend.post "/source/#{@project}/#{@package}?cmd=runservice&user=#{User.current}"
```

Comment:
BTW, this is something I hope in the future we will avoid having a Suse::Backend library that "knows" how to run a service, instead of having to hand craft a url for it and just sending a POST request.